Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.6-alpine
 
 ARG BUILD_DATE
 ARG VCS_REF
-ARG VERSION
+ARG VERSION=3.1.1
```

Comment:
This not needed, because actual version taken from VERSION file in root dir by Makefile.